bind-key.el

```diff
@@ -293,20 +293,17 @@ function symbol (unquoted)."
       (while (and cont args)
         (if (cond ((and (eq :map (car args))
                         (not prefix-map))
-                   (setq map (cadr args)))
+                   (let ((arg (cadr args)))
+                     (setq maps (if (listp arg) arg (list arg)))))
                   ((eq :prefix-docstring (car args))
                    (setq prefix-doc (cadr args)))
-                  ((and (eq :prefix-map (car args))
```

Comment:
I've removed this condition `(not (memq map '(global-map override-global-map)))` as, without understanding its purpose, I was not sure how to modify it to support the new list form of `:map`. Is there a reason why one could not use `:prefix-map` with the global maps?